gems/mail/OSVDB-131677.yml

```diff
@@ -16,4 +16,5 @@ description: |
   Recipient Email Addresses." 2015. The attacks described in the paper (Terada,
   p. 4) can be applied to the library without any modification.
 patched_versions:
-- ">= 2.6.0"
+- "~> 2.5.5"
+- ">= 2.6.6"
```

Comment:
`>= 2.5.5` is sufficient. 2.6.0-4 are unaffected due to a coincidental side effect of a separate change.